release_docs/README_HPC

```diff
@@ -145,19 +141,19 @@ cori, another CrayXC40, that line is replaced by "#SBATCH -C knl,quad,cache".
 For cori (and other machines), the values in LOCAL_BATCH_SCRIPT_NAME and
 LOCAL_BATCH_SCRIPT_PARALLEL_NAME in the config/cmake/scripts/HPC/sbatch-HDF5options.cmake
 file can be replaced by cori_knl_ctestS.sl and cori_knl_ctestS.sl, or the lines
-can be edited in the batch files in hdf5-1.13.0/bin/batch.
+can be edited in the batch files in hdf5-<version string(see section II)>/bin/batch.
```

Comment:
Intended, but agreed that it is odd.  The purpose was to let the user know how to find the version, but maybe it's unnecessary.  I can take it out if you think it would be better.

Reply:
I think the referral to section II from section I above is helpful, but here my first thought was just that it was a copy-paste issue. We could maybe still refer to section II somewhere in the sentence but I think it'd be better outside the path name.